wordpress-mobile/WordPress-iOS
# [Home Page Picker] Move to Page sheet

**Fixes** https://github.com/wordpress-mobile/WordPress-iOS/pull/15293#issuecomment-730520774

## To test:
<details>
<summary><strong>To disable or enable the development version of Home Page Picker</strong></summary>

- Open the app from the build that allows FeatureFlags such as a PR build or a local development build
    - By default, the modal layout picker will be disabled.
    - From the site page:
        - Click on your Gravatar.
        - Click on App Settings.
        - Click on Debug.
        - Toggle "Site Creation: Home Page Picker" to enable or disable the picker
        - <kdb><a href="https://user-images.githubusercontent.com/3384451/97343011-5f032980-185d-11eb-9030-06c17f732fec.gif"><img width="300" src="https://user-images.githubusercontent.com/3384451/97343011-5f032980-185d-11eb-9030-06c17f732fec.gif"/></a></kdb>

</details>

When enabled, the first step of the Site Creation flow will be replaced with the Home Page Picker flow.

### Test
1. Navigate to the Create WordPress.com site
    - My Sites > ➕ > Create WordPress.com site
1. **Expect** To see the site creation presented in a page sheet and the ✖️ to now say Cancel
1. Rotate to landscape
1. **Expect** To see the header collapse and cells to adjust for three columns
1. Tap Preview
1. **Expect** To see that presented in another page sheet similar to: https://cloudup.com/cBIAXJJMUcz and for the close button to be on the right

Complete the flow testing out landscape and portrait mode.

## Screenshots:
Step 1 | Preview
--- | ---
<kdb><a href="https://user-images.githubusercontent.com/3384451/99730144-f426c600-2a89-11eb-8645-918c3ddddb8d.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/99730144-f426c600-2a89-11eb-8645-918c3ddddb8d.png"/></a></kdb> | <kdb><a href="https://user-images.githubusercontent.com/3384451/99730156-f5f08980-2a89-11eb-86db-11fb68798a86.png"><img width="300" src="https://user-images.githubusercontent.com/3384451/99730156-f5f08980-2a89-11eb-86db-11fb68798a86.png"/></a></kdb>

## Additional Context: 

This is a continuation of the work done in the [Home Page Picker Project](https://github.com/wordpress-mobile/gutenberg-mobile/labels/Project%3A%20Home%20Page%20Picker). 

## PR submission checklist:
